CONTRIBUTING.md

```diff
@@ -129,6 +130,34 @@ We generally review for:
 To speed the process along, we may merge small changes such as spelling and formatting
 into your branch.
 Otherwise, we make suggestions and work with you to finalize the changes.
+<!-- vale Platform.first-person = YES -->
+
+### Review comment style
+
+To make comments clearer, each comment should have an emoji label describing its purpose.
+The following labels are used:
+
+| Emoji | `:code:`              | Meaning      | Description |
+| ----- | --------------------- | ------------ | ----------- |
+| 👍/😍 | `:+1:`/`:heart_eyes:` | Praise       | Used when something is well done. No further action required. Should be at least one of these for each review. |
+| ⛏️    | `:pick:`              | Nitpick      | For small details based on preferences. Doesn't block merging. |
+| 🛠️    | `:hammer_and_wrench:` | Suggestion   | To indicate an idea for improvement. By default, the suggestion needs to be accepted or otherwise addressed before merging. |
```

Comment:
❓ 💡 
As discussed during the weekly, maybe we could merge that one with the 💡 .
and add stuff like 🚫 or  ⚠️ for the blockers.
As you pointed out, it sounds "negative". 

Alternatively, we might just use the ❓ with any other emoji, to mark it as blocking?
So ❓ 💡  would be a blocking idea that needs discussion.
What do you think?

Reply:
I also think it's important that :bulb: represents a thought rather than an idea. An idea would be classified more as a suggestion.